content/src/view.js

```diff
@@ -1600,6 +1604,19 @@ function updatePaneTitle(pane) {
       label = 'data';
     } else if (/^text\/html/.test(paneState.mimeType)) {
```

Comment:
That will also fix the bug where `textonly = false` is getting missed - which causes you to currently lose the yellow tab block toggle button.